main/src/cgeo/geocaching/utils/CalculationUtils.java

```diff
@@ -96,13 +96,13 @@ boolean eat(final int charToEat) {
                     }
                     final String func = str.substring(startPos, this.pos);
                     x = parseFactor();
-                    if (func.equals("sqrt")) {
```

Comment:
I've changed it over to use a switch and it seems to compile fine.  It's not relay possible to check if it works the same though as these functions can never be reached in the current implementation.